### PR TITLE
packaging: add usage instructions for -a (arch_target) option

### LINKED ISSUE #534: build-kernel.sh  script  missing usage instructions for the -a  (arch_target)  option
https://github.com/kata-containers/kata-containers/blob/2.0-dev/tools/packaging/kernel/build-kernel.sh#L89
in  usage() function, the script missing -a option  instructions; same  problem  in  https://github.com/kata-containers/kata-containers/tree/2.0-dev/tools/packaging/kernel#usage

### PR DESCRIPTION
**Pull request by Kvasscn**
Add -a option in script usage function and README.

Fixes #534

Signed-off-by: Jason Zhang <zhanghj.lc@inspur.com>

### CODE REVIEW

**Comment by Tim-Zhang** [comment]
@Kvasscn All proposed commits should include a Signed-Off-By: <your-email-address> line in their commit message. This is most conveniently done by using --signoff (-s) when running git commit. 

Expected "zhanghj <zhanghj.lc@inspur.com>", but got "Jason Zhang <zhanghj.lc@inspur.com>

File: tools/packaging/kernel/build-kernel.sh [hunk 1]
```diff
@@ -88,16 +88,17 @@ Commands:
...
-	-d          : Enable bash debug.
-	-e          : Enable experimental kernel.
-	-f          : Enable force generate config when setup.
-	-g <vendor> : GPU vendor, intel or nvidia.
-	-h          : Display this help.
-	-k <path>   : Path to kernel to build.
-	-p <path>   : Path to a directory with patches to apply to kernel.
-	-t          : Hypervisor_target.
-	-v          : Kernel version to use if kernel path not provided.
+	-a <arch>   	: Arch target to build the kernel.
```
Comment (liubin) [inline tools/packaging/kernel/build-kernel.sh:91]: Adding https://github.com/kata-containers/kata-containers/blob/2.0-dev/tools/packaging/kernel/build-kernel.sh#L110-L113 as option candidate will be more helpful, like `-g <vendor>` options.

Comment (Kvasscn) [reply tools/packaging/kernel/build-kernel.sh:91]: ok, thanks. I recommit a patch later.

**Review by liubin — APPROVED** [review]

**Pull request merged**